src/routes/search/+page.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+  import "./search-page.scss";
+  import { onMount } from "svelte";
+  import algoliasearch from "algoliasearch";
+  import instantsearch from "instantsearch.js";
+  import { configure, hits, searchBox, stats, pagination } from "instantsearch.js/es/widgets";
+  import { searchHitsTemplate } from "./searchHelpers";
+
+  const ALGOLIA_APP_ID = "9IIDYROXZ5";
+  const ALGOLIA_API_KEY = "0eac885faaa4c70dfc9dd8b6ab4ab10f";
+  onMount(() => {
+    const search = instantsearch({
+      searchClient: algoliasearch(ALGOLIA_APP_ID, ALGOLIA_API_KEY),
+      // TODO: change this the "contentful" index when ready (name subject to change)
+      indexName: "media-sample-data",
+      routing: true,
```

Comment:
As long as the url + search query follows the [expected format of algolia](https://www.algolia.com/doc/guides/building-search-ui/going-further/routing-urls/js/#basic-urls), it will automatically populate the search with the correct search query upon initialization